### PR TITLE
Release/snowplow-web/0.4.1

### PR DESCRIPTION
**Pull request by bill-warner**
Couple of fixes for the web package.

### CODE REVIEW

File: packages.yml [hunk 1]
```diff
@@ -1,3 +1,3 @@
 packages:
-  - package: snowplow/snowplow_utils
-    version: [">=0.4.0", "<0.5.0"]
+  - git: "https://github.com/snowplow/dbt-snowplow-utils.git"
+    revision: "release/snowplow-utils/0.4.1"
```
Comment (paulboocock) [inline packages.yml:3]: Just a reminder to not forget to change this :-)

**Review by paulboocock — APPROVED** [review]

**Comment by bill-warner** [comment]
Realised by making the changes to the utils package that we break things for snowplow-web 0.4.0 users, as they would import the 0.4.1 utils package. I have updated both PRs to avoid this break. Once approved, I will release it on Monday now. Friday eve release makes me a bit uneasy

**Comment by colmsnowplow** [comment]
@bill-warner to confirm I understand the latest change - the break for the previous version is because `return_base_new_event_limits` (in the utils package)  would have returned an additional value which isn't expected in the previous version of the web package. 

So to resolve you factored the new value out into a new macro `get_session_lookback_limit`, and this version of the web package calls that macro to get that value, where the previous doesn't.

Is this correct?

**Review by colmsnowplow — APPROVED** [review]
Provided my understanding in this comment is correct, LGTM: https://github.com/snowplow/dbt-snowplow-web/pull/50#issuecomment-985569836

**Comment by bill-warner** [comment]
> @bill-warner to confirm I understand the latest change - the break for the previous version is because `return_base_new_event_limits` (in the utils package) would have returned an additional value which isn't expected in the previous version of the web package.
> 
> So to resolve you factored the new value out into a new macro `get_session_lookback_limit`, and this version of the web package calls that macro to get that value, where the previous doesn't.
> 
> Is this correct?

👍   Exactly that. Will probably integrate the `session_lookback_limit` returned from `get_session_lookback_limit` into the `return_base_new_event_limits` macro as part of v0.5. This just seemed like the easiest path for a patch release without a break.

**Pull request merged**